packages/docs/page-config/ui-elements/button/examples/Tag.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <va-button
-    class="mr-6"
+    class="mr-6 mb-2"
```

Comment:
Grid solution would be better, because it removes extra margin here:

<img width="844" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/1c5803d3-79a5-49ce-a459-6818c7e8a19b">

But we still need to wrap button with div, so button looks smaller as they look right now.